packages/studio/src/components/PreviewWithUseComponents.tsx

```diff
@@ -11,6 +11,10 @@ const inlineStyles: CSSProperties = {
   overflow: "scroll",
 };
 
+const tooltipStyle: CSSProperties = {
+  zIndex: "20",
```

Comment:
can we add the `z-20` tailwind class to the tooltip `className` instead of adding a `style`? generally, we prefer to use tailwind classes for styling where possible

Reply:
oh also switched it to z-30 as panels had 20

Reply:
which panels have z-20? I thought only our tooltips would have that index

Reply:
Ah you are right. I misread it, sorry.

Reply:
no worries! it's not technically a problem, but I just find it easier and cleaner to only have a couple z-indexes to keep track of